src/board-explorer-app/board-explorer-app.html

```diff
@@ -436,14 +436,36 @@
       this.clearPin();
 
       var index = event.detail.value.getAttribute('index');
-      var filterList = this.categories[index].list;
+      var category = this.categories[index];
 
       /* Only add items to the filter if they exist on the board */
       this.components = this.availableComponents.filter(function(component) {
-        return (filterList.indexOf(component.refdes) != -1) ? true : false;
+        return (category.list.indexOf(component.refdes) != -1) ? true : false;
       });
 
-      this.$.boardViewer.addFilter("selected", filterList, {
+       /* Only add keywords to the filter if they exist on the board */
+      this.keywords = this.availableKeywords.filter(function(keyword) {
+        return (category.list.indexOf(keyword.keyword) != -1) ? true : false;
+      });
+
+      var href = window.location.href.replace(/#.*/, ""); /* Trim of # */
+      /* If the board is set, add it to the URL */
+      if (this.board) {
+         href += "#" + this.board;
+      }
+      /* If the filters are set, add them to the URL */
+      if (category.href) {
```

Comment:
I'm adding this property since the board-viewer. so it will need a PR from there.